src/nodes.coffee

```diff
@@ -1223,18 +1223,128 @@ exports.Class = class Class extends Base
     klass = new Assign @variable, klass if @variable
     klass.compileToFragments o
 
+#### Import and Export
+
+exports.Module = class Module extends Base
+  constructor: (@type, @clause, @moduleName, @default = no) ->
+    if @type isnt 'import' and @type isnt 'export'
+      @error 'module type must be import or export'
+
+    if @moduleName? and @moduleName instanceof StringWithInterpolations
+      @error 'the name of the module to be imported from must be an uninterpolated string'
+
+    if @type is 'export' and @default is no and @clause instanceof Class and not @clause.variable?
+      @error 'anonymous classes cannot be exported'
+
+  children: ['clause', 'moduleName']
+
+  isStatement: YES
+  jumps:       THIS
+  makeReturn:  THIS
+
+  compileNode: (o) ->
+    if o.indent.length isnt 0
+      @error "#{@type} statements must be at top-level scope"
```

Comment:
Meteor wants code blocks like this:

```js
if (Meteor.isServer) {
  import { foo } from 'lib';
}
```

which of course violates the spec, but their build tool somehow parses this. I guess we should support such a syntax, and let the user violate the “`import` or `export` statements must be at top level” requirement if they choose? If we allow this, how can I rewrite the other places where I look at `o.indent` to use some other way of determining if we’re in an `export` statement, rather than an expression within an `export` statement?